Source/ImPlotWidgets/Private/LineFilledGraphWidget.cpp

```diff
@@ -51,6 +45,18 @@ void SFilledGraphWidget::OnImGuiDraw() const
 	constexpr int32 GraphFlags = ImPlotFlags_AntiAliased;
 	constexpr int32 AxisFlags = ImPlotAxisFlags_NoGridLines;
 
+	TArray<float> VisibleLineMaxValues;
+
+	TArray<TArray<float>> FittedYValues;
```

Comment:
After further investigation it would be better to not do this refitting and cache the data, ImGui has functions to apply to projection/getter across arbitrary data on draw, which would allow us to convert the data only on draw when needed.